engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetFacesToCull.java

```diff
@@ -37,7 +37,9 @@
     private int mode;
 
     public SetFacesToCull(int mode) {
-        if (!modeNameMap.containsKey(mode)) {
+        if (mode != GL_BACK
```

Comment:
I was getting NPE, because of uninitialised static variable during initialisation. I didn't have the time to properly check why this tiny change resulted as NPE, so I reverted it back. For if-block, let me correct that.